release-1.6/go-signal-reports/2017-03-13.md

```diff
@@ -0,0 +1,205 @@
+- [k8s.io] Network Partition [Disruptive] [Slow] [k8s.io] [ReplicationController] should eagerly create replacement pod during network partition when termination grace is non-zero 
+- [k8s.io] Services should be able to change the type and ports of a service [Slow]
+- [k8s.io] Services should only allow access from service loadbalancer source ranges [Slow]
+- [k8s.io] Generated release_1_5 clientset should create pods, delete pods, watch pods
+- [k8s.io] Namespaces [Serial] should ensure that all pods are removed when a namespace is deleted.
+- [k8s.io] NoExecuteTaintManager [Serial] eventually evict pod with finite tolerations from tainted nodes
+
+### Action Required
+- Please address all failing tests
+- Please add SIG routing labels to issued opened due to test failure
+
+## Suite Failures
+- [Federation, GCE](https://k8s-testgrid.appspot.com/google-federation#gce)
+- [Federation, GCI-GKE](https://k8s-testgrid.appspot.com/google-federation#gci-gce)
+- [Node, heapster push](https://k8s-testgrid.appspot.com/google-node#heapster-push)
+- [1.5 -> 1.6 GCI kubectl skew](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gce-1.5-1.6-gci-kubectl-skew)
+- [1.5 -> 1.6 Container VM kubectl skew](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gce-1.5-1.6-cvm-kubectl-skew)
+- [GKE 1.5 -> 1.6 Container VM kubectl skew](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gke-1.5-1.6-cvm-kubectl-skew)
+- [GKE 1.5 -> 1.6 GCI kubectl skew](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gke-1.5-1.6-gci-kubectl-skew)
+- [GCE 1.5 -> 1.6 upgrade cluster](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gce-1.5-1.6-upgrade-cluster)
+- [GCE 1.5 -> 1.6 upgrade cluster new](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gce-1.5-1.6-upgrade-cluster-new)
+- [GCE 1.5 -> 1.6 upgrade master](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gce-1.5-1.6-upgrade-master)
+- [GKE 1.5 -> 1.6 upgrade cluster](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gke-container_vm-1.4-container_vm-1.6-upgrade-cluster)
+- [GKE 1.5 -> 1.6 GCI upgrade cluster](https://k8s-testgrid.appspot.com/release-1.6-upgrade-skew#gke-gci-1.5-gci-1.6-upgrade-cluster)
+- [GKE Serial 1.6](https://k8s-testgrid.appspot.com/release-1.6-blocking#gke-serial-1.6)
+- [GKE GCI Serial 1.6](https://k8s-testgrid.appspot.com/release-1.6-blocking#gci-gke-serial-1.6)
+- [GCE Federation 1.6](https://k8s-testgrid.appspot.com/release-1.6-blocking#gce-federation-1.6)
+- [GCE Multizone](https://k8s-testgrid.appspot.com/google-gce#gce-multizone)
+
+### Action Required
+- GKE Suite Owner must be identified
+- GCE Suite Owner must be identified
+- Fix GCE skew testing
+- Fix GKE skew testing
+- Fix Federation suites
+
+## Incomplete Features
+- Open issues filed against [`kubernetes/features` in 1.6 milestone](https://github.com/kubernetes/features/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20milestone%3Av1.6): 31 (**unchanged from last report**)
```

Comment:
#0 features (per [features repo](https://github.com/kubernetes/features/issues?q=is%3Aissue+is%3Aopen+milestone%3Av1.6) and [spreadsheet](https://docs.google.com/spreadsheets/d/1nspIeRVNjAQHRslHQD1-6gPv99OcYZLMezrBe3Pfhhg/edit?usp=sharing))